Hymns/SearchView.swift

```diff
@@ -0,0 +1,30 @@
+import SwiftUI
+
+struct SearchView: View {
+    var hymns = hymnTestData
+    @State private var searchText: String = ""
+
+    //Currently we have to use UIKIT to change the unselected tabicons from grey to black.
+    init() {
+        UITabBar.appearance().unselectedItemTintColor = UIColor.black
+    }
+    
+    var body: some View {
+        NavigationView {
+            VStack {
+                SearchBar(text: $searchText)
+                List {
+                    ForEach(hymns.filter { self.searchText.isEmpty ? true : $0.contains(self.searchText)}, id: \.self) { hymn in
+                        Text(hymn)
+                    }
+                }.navigationBarTitle(Text("Look up any hymn"))
```

Comment:
If it's not too much trouble, could we put these in some kind of localized file? Instead of hardcoding the strings?

Reply:
I went ahead and add the localized files. So the way they work is quite nice. We still leave this hardcoded but it will automatically convert to the right text based on what we define in our localized string files.